Firestore/core/src/firebase/firestore/local/leveldb_persistence_posix.cc

```diff
@@ -20,8 +20,9 @@
 #include "Firestore/core/src/firebase/firestore/util/path.h"
 #include "Firestore/core/src/firebase/firestore/util/status.h"
 
-#if !defined(__APPLE__) && !defined(_WIN32)
+#if !__APPLE__ && !_WIN32
 
+#include <asm-generic/errno-base.h>
```

Comment:
This is the wrong header. This should be the C++ header `<cerrno>`.

Reply:
Done.